remix-auth/app/components/auth/ResendVerificationEmail.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { auth } from "~/services/auth.server";
+
+export default function ResendVerificationEmail({
+  verificationToken,
+}: {
+  verificationToken: string;
+}) {
+  const [sent, setSent] = useState(false);
+  const [sending, setSending] = useState(false);
+
+  return sent || sending ? (
+    <div className="text-slate-600 mt-2">
+      {sent ? "Verification email sent!" : "Sending verification email..."}
+    </div>
+  ) : (
+    <button
+      type="button"
+      onClick={async () => {
+        setSending(true);
+        await auth.emailPasswordResendVerificationEmail({
```

Comment:
This won't work, you can't call a method on the server auth object directly in a component.

Reply:
yes tnx, missed it